### PR TITLE
Add $device_type tooltip text

### PR DESCRIPTION
**Pull request by mariusandra**
## Changes

- Companion to PRs: https://github.com/PostHog/posthog-js/pull/198 and https://github.com/PostHog/plugin-server/pull/228
- Adds nice text about the `$device_type` and `$initial_device_type` properties.

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests

### CODE REVIEW

File: frontend/src/lib/components/PropertyKeyInfo.js [hunk 1]
```diff
@@ -163,10 +163,15 @@ export const keyMapping = {
             description: 'Direct link to the exception in Sentry',
             examples: ['https://sentry.io/...'],
         },
-        $device: {
-            label: 'Device',
-            description: 'The mobile device that was used.',
-            examples: ['iPad', 'iPhone', 'Android'],
```
Comment (Twixes) [inline frontend/src/lib/components/PropertyKeyInfo.js:169]: I don't think we should remove this. Also, IMO we should support `$initial_device` too, as an analogy (https://github.com/PostHog/plugin-server/pull/228/#discussion_r586238960).

Comment (mariusandra) [reply frontend/src/lib/components/PropertyKeyInfo.js:169]: It was duplicated

Comment (mariusandra) [reply frontend/src/lib/components/PropertyKeyInfo.js:169]: image

Comment (Twixes) [reply frontend/src/lib/components/PropertyKeyInfo.js:169]: Alright, thanks

**Pull request merged**